src/Analyzers/VisualBasic/CodeFixes/RemoveUnnecessaryCast/VisualBasicRemoveUnnecessaryCastCodeFixProvider.vb

```diff
@@ -54,8 +50,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.RemoveUnnecessaryCast
         End Function
 
         Protected Overrides Async Function FixAllAsync(
-            document As Document, diagnostics As ImmutableArray(Of Diagnostic),
-            editor As SyntaxEditor, cancellationToken As CancellationToken) As Task
+document As Document,             diagnostics As ImmutableArray(Of Diagnostic),
+editor As SyntaxEditor, options As CodeActionOptionsProvider, cancellationToken As CancellationToken) As Task
```

Comment:
something weird happened :)

Reply:
Seems like a bug in update signature refactoring.